src/diffusers/pipelines/animatediff/pipeline_animatediff.py

```diff
@@ -555,6 +585,8 @@ def __call__(
         callback_steps: Optional[int] = 1,
```

Comment:
IIRC, `callback` and `callback_steps` has been deprecated in favour of `callback_on_step_end` and `callback_on_step_end_tensor_inputs`.